templates/support.rackspace.com/home.html

```diff
@@ -1,5 +1,12 @@
 {% extends "_layouts/base.html" %}
 
+
+## ALEX DOESNT KNOW WHAT SHE IS DOING HERE
```

Comment:
The header is stored on a CDN, actually. I think it's a relic of the self-service days; it actually used to be stored up in enterprise in a hidden repo (that was fun to find, believe me). We're supposed to be redesigning the site at some point anyway, so I haven't really taken the time to move it here :/ I do agree with you, though.